server/redeemer.go

```diff
@@ -129,22 +132,28 @@ func (r *Redeemer) MonitorMaxFloat(req *net.MaxFloatReq, stream net.TicketRedeem
 			return nil
 		case <-stream.Context().Done():
 			return nil
+		case err := <-errCh:
```

Comment:
Ah I see. In that case, keeping the current behavior for now seems fine.

> But I definitely don't have the whole picture of who is calling this RPC call and why.

The caller for this RPC method would be an O that is outsourcing ticket redemption to a node that is running a standalone Redeemer service. Normally, an O would monitor the max float [1] of Bs that are sending it tickets (i.e. payments) with a stream. If an O is outsourcing ticket redemption to a Redeemer then an O will subscribe to max float updates for Bs that are published by the Redeemer which becomes the one responsible for monitoring the max float of Bs. 

[1] Think of the "max float" as the maximum sum of value across winning tickets from B that O is willing to hold because that value is guaranteed to be available in B's on-chain reserve funds. See https://github.com/livepeer/wiki/blob/master/spec/streamflow/pm.md#reserve for more details.

Reply:
Ahh, I see. I didn't even know that we can run Livepeer as a standalone Redeemer service! Thanks for the explanation :)